sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -265,4 +266,41 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       };
     });
   }
+
+  /**
+   * Gets all eligible application that can be requested for application
+   * offering change.
+   * @param locationId location id.
+   * @param paginationOptions options to execute the pagination.
+   * @returns list of eligible application that can be requested for
+   * application offering change.
+   */
+  @HasLocationAccess("locationId")
+  @Get(":locationId/active-applications/request-change")
+  async getEligibleApplicationOfferingChangeRecords(
+    @Param("locationId", ParseIntPipe) locationId: number,
+    @Query() pagination: ApplicationStatusPaginationOptionsAPIInDTO,
+  ): Promise<
+    PaginatedResultsAPIOutDTO<ApplicationOfferingChangeSummaryAPIOutDTO>
+  > {
+    const applications =
+      await this.applicationService.getEligibleApplicationOfferingChangeRecords(
+        locationId,
+        pagination,
+      );
+
+    return {
+      results: applications.results.map((eachApplication) => {
+        const offering = eachApplication.currentAssessment?.offering;
+        return {
+          applicationNumber: eachApplication.applicationNumber,
+          applicationId: eachApplication.id,
+          studyStartPeriod: getISODateOnlyString(offering?.studyStartDate),
```

Comment:
Offering `studyStartDate` and `studyEndDate` are already date-only columns that should be ready to be returned by the API and they are without the need to call `getISODateOnlyString`.